src/commands/buildApp.ts

```diff
@@ -96,7 +96,16 @@ export async function buildApplication (node: DeviceNode | OSVerNode | PlatformN
 		await vscode.tasks.executeTask(task);
 	} catch (error) {
 		if (error instanceof InteractionError) {
-			await handleInteractionError(error);
+			await handleInteractionError(error)
+				.then(async function () {
+					const choice = await vscode.window.showErrorMessage('Build App failed', { title: 'Rerun' });
+					if (!choice) {
+						return;
+					}
+					if (choice.title === 'Rerun') {
```

Comment:
Now that we can use optional chaining we could maybe use that here to avoid needing the 2 if's
```suggestion
					if (choice?.title === 'Rerun') {
```